### DISCUSSION TITLE
gbuffers limitation?

### DISCUSSION THREAD

**Discussion by jebbyk** [Open4Es/Open4Es-Shader-Android#61] [Q&A]
I cant write to additional buffers (for example I wanna pass normals and depth from gbuffers_terrain into composite shader). On my phone with Optifine HD G6 Ultra Im getting error (without any details). On my PC with Optifine HD G8_pre2 Ultra it works fine. Is it a limitation of Oprifine G6 or a limitation of Pojav?

**Comment by ENDERMANYK** [comment]
it caused by GL4ES, it currently only support one drawbuffer
